src/ngx_http_lua_ssl_ocsp.c

```diff
@@ -383,6 +384,16 @@ ngx_http_lua_ffi_ssl_validate_ocsp_response(const u_char *resp,
         goto error;
     }
 
+    if (nextupdate) {
```

Comment:
Shall we explicitly test if `nextupdate` is equal to zero instead of testing it as a boolean value?

Reply:
It's a double pointer to `ASN1_GENERALIZEDTIME` type, I don't know if we can easily compare it to zero. Also NGINX follows this approach: https://github.com/nginx/nginx/blob/7547581bbcb7b737710f9141260d822a08685b83/src/event/ngx_event_openssl_stapling.c#L2392